extensions/notebook/src/book/bookVersionHandler.ts

```diff
@@ -56,6 +56,15 @@ export function convertFrom(version: string, section: JupyterBookSection): Jupyt
 	}
 }
 
+function setNotNumbered(section: JupyterBookSection): boolean | undefined {
```

Comment:
I'd suggest a different name for this (or change the logic). Currently it's not actually setting anythingzAAAAA